src/analysis/normed_space/basic.lean

```diff
@@ -1607,6 +1614,30 @@ theorem frontier_closed_ball [semi_normed_space ℝ E] (x : E) {r : ℝ} (hr : 0
 by rw [frontier, closure_closed_ball, interior_closed_ball x hr,
   closed_ball_diff_ball]
 
+theorem smul_ball_0 {c : α} (hc : c ≠ 0) (r : ℝ) :
```

Comment:
Don't we normally use `zero` rather than `0` in lemma names?

Reply:
Alll the lemmas in this file mentioning balls centered at 0 use 0, but I can rename all of them.

Reply:
I'll leave it up to you

Reply:
I have renamed all of them to `zero`.